autocodesign/autocodesign.go

```diff
@@ -110,6 +112,18 @@ type CertificateProvider interface {
 // LocalCertificates is a map from the certificate type (development, distribution) to an array of installed certs
 type LocalCertificates map[appstoreconnect.CertificateType][]certificateutil.CertificateInfoModel
 
+// LocalProfile ...
+type LocalProfile struct {
+	APIProfile Profile
+	Info       profileutil.ProvisioningProfileInfoModel
+}
+
+// ProfileProvider returns manually generated profiles, used as a fallback
```

Comment:
why manually generated profiles only? I think how other code uses this interface should not be here.

Reply:
Made less specific.